eng/packages/http-client-csharp-mgmt/generator/TestProjects/Local/Mgmt-TypeSpec/foo.tsp

```diff
@@ -59,7 +59,7 @@ model FooSettings is ProxyResource<FooSettingsProperties> {
 @armResourceOperations
 interface FooSettingsOperations {
   get is ArmResourceRead<FooSettings>;
-  createOrUpdate is ArmResourceCreateOrReplaceSync<FooSettings>;
+  createFooSettings is ArmResourceCreateOrReplaceSync<FooSettings>;
```

Comment:
One example in the spec would be enough, we could change this or the method name in Zoos back.
`createOrUpdate` is a more typical pattern